### PR TITLE
Feat/twi new cards

### PR DESCRIPTION
**Pull request by mariotee**
- Soulless One
- In Defense of Kimino
- Pyrrhic Assault

### CODE REVIEW

File: AllyAbilities.php [hunk 1]
```diff
@@ -352,6 +353,18 @@ function DestroyAlly($player, $index, $skipDestroy = false, $fromCombat = false,
   return $cardID;
 }
 
+function CurrentEffectsContainWhenDefeated($player) {
+  global $currentTurnEffects;
+  for($i=0;$i<count($currentTurnEffects); $i+=CurrentTurnEffectPieces()) {
+    switch($currentTurnEffects[$i]) {
+      case "1272825113":
```
Comment (brubraz) [inline AllyAbilities.php:360]: Could you please add a comment with the name of the card? It's really helpful for maintenance.

Comment (mariotee) [reply AllyAbilities.php]: thank you that is now udpated. also worked for me to remove the webp files from the Unimplemented folder for those two

File: AllyAbilities.php [hunk 2]
```diff
@@ -740,10 +754,27 @@ function AllyDestroyedAbility($player, $cardID, $uniqueID, $lostAbilities,
         AddDecisionQueue("MZOP", $player, "ADDHEALTH,2", 1);
         AddDecisionQueue("MZOP", $player, "GETUNIQUEID", 1);
         AddDecisionQueue("ADDLIMITEDCURRENTEFFECT", $player, "0249398533,PLAY", 1);
-        break;
+        break;
```
Comment (VJubert) [inline AllyAbilities.php:757]: revert this line

Comment (mariotee) [reply AllyAbilities.php]: updated

**Pull request merged**